examples/ITK_Example_RegistrationWithCorrespondingPointsInMemory.ipynb

```diff
@@ -0,0 +1,529 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #60.        ax.imshow(cmap="binary_r", origin="lower", X=itk.GetArrayViewFromImage(image))
</pre><p>itk.array_view_from_image is preferred over itk.GetArrayViewFromImage for style.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks! Should we eventually deprecate itk.GetArrayViewFromImage in favor of itk.array_view_from_image ?